feat(tauri) read templates from @quasar/tauri package

lucasfernog opened the pull request.

rstoenescu committed:
feat(docs): Update sponsors

rstoenescu committed:
feat(docs): mobile-apps links should be cordova-apps due to forthcoming capacitor mode

rstoenescu committed:
fix(docs): typo in code example on notify page

rstoenescu committed:
feat(docs): Updated sponsors and backers page

rstoenescu committed:
fix(app): [ios] using ionic wkwebview or original wkwebview errors out when enabling HTTPS #4551

rstoenescu committed:
chore: Bump version

wtoalabi committed:
Fixed a typo (#4568)

rstoenescu committed:
feat(docs): Small tweak to one QSelect example

rstoenescu committed:
feat(Platform): Detect mobile edge #4573

rstoenescu committed:
feat(docs): Improve native attributes description for QInput and QSelect #4561

rstoenescu committed:
feat(Platform): add $q.platform.userAgent

rstoenescu committed:
feat(docs): Update Platform Detection page

rstoenescu committed:
feat(Platform): Improve mobile platform detection

pdanpdan committed:
fix(QSelect): Improve performance for initializing heights (#4579)

pdanpdan committed:
fix(QResizeObserver): Use void 0 to reset curDocView (#4589)

close #4588

rstoenescu committed:
feat(docs): Update sponsor list

pdanpdan committed:
fix(VTouchPan/Swipe): Prevent dragging of a and img in Firefox (#4590)

* fix(VTouchPan/Swipe): Prevent dragging of a and img in Firefox

close #4585

* Update event.js

* Update event.js

* Update event.js

* Update TouchPan.js

* Update TouchPan.js

* Update TouchSwipe.js

rstoenescu committed:
chore(ui): fix linting issue

rstoenescu committed:
chore(docs): remove arrow on landing page

mesqueeb committed:
update title for better contrast with first section (#4597)

rstoenescu committed:
fix(docs): Upgrade page headings

outofmemoryagain committed:
fix: #4411 Add tsName for Dialog Options (#4598)

* Support tsType in type generation script for method parameters, and non-function properties.

Modify Dialog.json create method to opts arguement to have tsName: "QDialogOptions"

* Properly pass required property when generating method arguments for generated named Type interfaces

pdanpdan committed:
fix(QInput/QSelect): Prevent composition on android when codepoint is below 256 (#4591)

close #4578

rstoenescu committed:
feat(docs): Slightly improve landing page

rstoenescu committed:
fix(app): App Extension - Install API - render interpolation #4586

rstoenescu committed:
chore(app): Bump version

rstoenescu committed:
chore(ui): Bump version

rstoenescu committed:
chore(docs): update deps

rstoenescu committed:
feat(docs): add fb link; use quasar.dev subdomains

rstoenescu committed:
fix(docs): quasar cli > cordova related pages are not accessible

fpmar committed:
Add ASP.NET Web API example. (#4605)

rstoenescu committed:
feat(docs): polish docs on asp.net support for QUploader #4605

valasek committed:
feat(docs) Contribution guide added (#4525)

* Contribution guide added

* docs: updated broken links

* docs: upgrade guide reverted QColorPicker

* Update contribution-guide.md

* fix(docs): QColor link fixed

* fix(docs): QColorPicker heading fixed

* fix(docs): QColorPicker heading fixed

rstoenescu committed:
feat(docs): polish contribution guide page

wtoalabi committed:
Few word fixes (#4592)

* Removed extra word

* Re-arranged some words

* Not a big deal, can be ignored.

I just felt that the word does not resonate with the day to day meaning of its contextual usage.
Some experts believed that the word "existent" is more common in religious and philosophical writings. 
But again, not too much of a big deal. The consumers of this doc could not care less, I suppose.

* Changed to 'an' article

rstoenescu committed:
feat(app): Avoid copying /statics/icons/* and app-logo-128x128.png for Cordova/Electron because they serve no purpose

mdpx committed:
Update umd.md (#4607)

nothingismagick committed:
fix(docs): typo on introduction to quasar page (#4610)

minor fix

rstoenescu committed:
fix&perf(QDrawer): Safari doesn't render correctly the ease-in animation #4611

nothingismagick committed:
fix(docs): typo in ssr faq page (#4618)

rstoenescu committed:
feat(docs): update newsletter page

pdanpdan committed:
fix(Screen): Properly detect screen.is.xl (#4623)

close #4622

pdanpdan committed:
fix(quasar): Fix $sizes usage in variables.styl (#4620)

dependabot[bot] committed:
chore(deps): bump lodash.template from 4.4.0 to 4.5.0 in /ui (#4624)

Bumps [lodash.template](https://github.com/lodash/lodash) from 4.4.0 to 4.5.0.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.4.0...4.5.0)

Signed-off-by: dependabot[bot] <support@github.com>

rstoenescu committed:
feat(app): Update deps

rstoenescu committed:
chore(babel-preset-app): update yarn.lock

rstoenescu committed:
chore(app): update yarn.lock

rstoenescu committed:
chore(app): Bump version

rstoenescu committed:
chore(ui): update deps

rstoenescu committed:
chore(cli): update yarn.lock

pdanpdan committed:
fix(QTab): Allow anchor related components to work inside QTab (#4626)

close #4625

vreeken committed:
Fix spelling mistake (#4632)

intitialization -> initialization

rstoenescu committed:
feat(extras): Add Material Icons - outlined, round & sharp flavours #4635

rstoenescu committed:
feat(QIcon): Support for Material Icons Outlined/Round/Sharp #4635

rstoenescu committed:
feat(docs): Add instructions for Material Icons Outlined/Round/Sharp #4635

rstoenescu committed:
feat(docs): Small update to header menu

rstoenescu committed:
feat(docs): Add @quasar/qribbon to official App Extensions list

rstoenescu committed:
fix(QDrawer): Drawer gets stuck in certain scenarios on mobile platforms #4614

pdanpdan committed:
fix(QSlideTransition): Workaround transitionend not firing when animated element is hidden (#4638)

ref #4630

outofmemoryagain committed:
Add Quasar Plugin Type for use with Quasar Testing extension. (#4634)

rstoenescu committed:
feat(QDrawer): Proxy mouseenter and mouseleave events #4639

valasek committed:
chore(docs) - share tech stack added (#4643)

outofmemoryagain committed:
Change Quasar Plugin options to support only passing some options. (#4645)

sskwrl committed:
initial version of vs config page (#4633)

rstoenescu committed:
chore(ui): Bump version

rstoenescu committed:
chore(docs): Update deps

rstoenescu committed:
feat(docs): Add tutorial link

metalsadman committed:
Update upgrade-guide.md (#4647)

* Update upgrade-guide.md

Back to top directive wasn't mentioned.

* Update upgrade-guide.md

hawkeye64 committed:
fix(docs): spelling (#4644)

Someone92 committed:
Update vs-code-configuration.md (#4649)

Removed quotation mark from boolean in VS Code settings

lucasfernog committed:
feat(proton) read templates from @quasar/proton package

nothingismagick commented:
The templates are located at @quasar/proton/templates

Justification for this is as follows:
- It allows us to iterate quickly
- It keeps core "pure" from non-node components
- Enables easy integration of other bindings

i.e. A separation of concerns.

Please see: https://github.com/quasarframework/proton/tree/dev/templates

rstoenescu commented:
The reason I asked for separate proton repo is to use it for forking cargo packages.
The template that gets into the user space for proton should stay in the CLI (I'll be regularly checking PRs to it and merging immediately during my vacation). 
We shouldn't create a (possibly temporary) @quasar/proton package to wrap cargo packages. Instead, use the "git" approach for these packages within the template from CLI.

pdanpdan committed:
fix(docs): Fix size of hero container and Quasar button on IE

mesqueeb committed:
problem with double usage of single line ` (#4662)

* problem with double usage of single line `

* Update upgrade-guide.md

mesqueeb committed:
move border related css to border related sub-group (#4663)

mesqueeb committed:
wrong formatting for table (#4669)

rstoenescu committed:
Update Landing.vue

rstoenescu committed:
Merge pull request #4661 from pdanpdan/v1-docs-landing

fix(docs): Fix size of hero container and Quasar button on IE

rstoenescu committed:
feat(api): Improve Screen plugin API description #4668

pdanpdan committed:
fix(QStepper): Use brand color in QStepper CSS

rstoenescu committed:
Merge pull request #4674 from pdanpdan/v1-stepper-css-variables

fix(QStepper): Use brand color in QStepper CSS

pdanpdan committed:
feature(docs): Explain the special case of QDate, QTime, ... with QPopupProxy

ref #4230

rstoenescu committed:
Merge pull request #4676 from pdanpdan/v1-docs-popup-proxy-special

feature(docs): Explain the special case of QDate, QTime, ... with QPopupProxy

rstoenescu committed:
Update Basic.vue

rstoenescu committed:
Update Styled.vue

proofoftom committed:
Fix export of right drawer as overlay

Is currently using left overlay settings and doesn't export right overlay directive as expected

lucasfernog committed:
feat(proton) serverless & bundle config

rstoenescu committed:
Merge pull request #4707 from proofoftom/patch-1

Fix export of right drawer as overlay

lucasfernog committed:
feat(proton) inline all assets on serverless mode

lucasfernog committed:
feat(proton) clear only compiled-web/ build artifact

lucasfernog committed:
feat(proton) rename entry class to Proton

lucasfernog committed:
fix(quasar) proton dev mode when serverless = true

lucasfernog committed:
feat(proton) use proton-bundle for bundling Rust code

Commit:
feat(whitelist) introduce whitelisting

Signed-off-by: Daniel Thompson-Yvetot <denjell@quasar.dev>

lucasfernog committed:
feat(app) whitelist proton API on Rust lib

Commit:
feat(whitelist) Object.freeze and JSDocs

Signed-off-by: Daniel Thompson-Yvetot <denjell@quasar.dev>

Commit:
Merge remote-tracking branch 'origin/feature/whitelisting' into feature/whitelisting

lucasfernog committed:
Merge pull request #1 from lucasfernog/feature/whitelisting

feat(whitelist) Add whitelist

Commit:
fix(app:proton) can't freeze it if it doesn't really exist

lucasfernog committed:
fix(app) wrong feature name for execute command

lucasfernog committed:
fix(whitelist) conflicting names

pdanpdan committed:
fix(ie): Restrict QField's label width in IE

Elements occupy the width before scale in IE

rstoenescu committed:
Merge pull request #4722 from pdanpdan/v1-ie-field-label-width

fix(ie): Restrict QField's label width in IE

lucasfernog committed:
feat(proton) open urls

lucasfernog committed:
feat(app) copy proton config to @quasar/proton package

mike-feldmeier committed:
Correct reference to unmasked-value

rstoenescu committed:
Merge pull request #4738 from mike-feldmeier/patch-1

Correct reference to unmasked-value

Commit:
feat(proton) link proton to /quasar/proton

MechJosh0 committed:
added aria role to q-tooltip

lucasfernog committed:
Merge pull request #2 from lucasfernog/feat/move-proton-gitsource

feat(proton) link proton to /quasar/proton

rstoenescu committed:
Merge pull request #4743 from MechJosh0/patch-1

feat(QTooltip): Added aria role

Commit:
fix(stupidity) wrong link

lucasfernog committed:
Merge branch 'dev' into feature/proton-package

lucasfernog committed:
feat(app) inject eventHandlers

lucasfernog committed:
feat(proton) addEventListener on backend layer

lucasfernog committed:
feat(proton-entry) callbacks from event answer response

lucasfernog committed:
feat(event-system) simplify

lucasfernog committed:
feat(proton) better event interface

lucasfernog committed:
chore(app) remove debug statement

lucasfernog committed:
test

lucasfernog committed:
Revert "test"

This reverts commit 6becd514b48c50e1cd3632b1d7d876413e946ee3.

lucasfernog committed:
Merge remote-tracking branch 'origin/feature/proton-package' into feature/proton-config

lucasfernog committed:
feat(proton-config) default window title from package.json

lucasfernog committed:
feat(proton-config) bundle as object

lucasfernog committed:
feat(proton-config) rename serverless to embeddedServer

lucasfernog committed:
feat(proton-config) embedded server host and port config

lucasfernog committed:
feat(proton-config) CSP config

lucasfernog committed:
chore(quasar) rename proton to tauri

lucasfernog committed:
Merge remote-tracking branch 'origin/feature/proton-config' into feature/proton-package

lucasfernog committed:
feat(app) remove cargo --bin arg

lucasfernog committed:
fix(app) remaining proton names

lucasfernog committed:
feat(tauri) use CLI lib

lucasfernog committed:
Merge pull request #3 from lucasfernog/feature/tauri-cli

feat(tauri) use CLI lib
